packages/app/src/systems/Bridge/pages/BridgeTxList.tsx

```diff
@@ -28,94 +27,60 @@ export const BridgeTxList = () => {
-            onClick={handlers.connect}
-          />
-        )}
-        {shouldShowEmpty && <BridgeListEmpty />}
-        {shouldShowList && (
-          <CardList isClickable>
-            {bridgeTxs.map((txDatum, index) => {
-              if (
-                isEthChain(txDatum.fromNetwork) &&
-                isFuelChain(txDatum.toNetwork)
-              ) {
-                return (
-                  <TxListItemEthToFuel
-                    key={`${index}-${txDatum.txHash}`}
-                    txHash={txDatum.txHash || ''}
-                    asset={txDatum.asset}
-                    isDone={txDatum.isDone}
-                  />
-                );
-              }
-              if (
-                isFuelChain(txDatum.fromNetwork) &&
-                isEthChain(txDatum.toNetwork)
-              ) {
-                return (
-                  <TxListItemFuelToEth
-                    key={`${index}-${txDatum.txHash}`}
-                    txHash={txDatum.txHash || ''}
-                    asset={txDatum.asset}
-                    isDone={txDatum.isDone}
-                    date={txDatum.date}
-                  />
-                );
-              }
+    <>
+      {isLoading && <BridgeTxItemsLoading />}
+      {shouldShowNotConnected && (
+        <BridgeTxListNotConnected
+          isConnecting={isConnecting}
+          onClick={handlers.connect}
+        />
+      )}
+      {shouldShowEmpty && <BridgeListEmpty />}
+      {shouldShowList && (
+        <CardList isClickable css={styles.cardList}>
+          {bridgeTxs.map((txDatum, index) => {
+            if (
+              isEthChain(txDatum.fromNetwork) &&
+              isFuelChain(txDatum.toNetwork)
+            ) {
+              return (
+                <TxListItemEthToFuel
+                  key={`${index}-${txDatum.txHash}`}
+                  txHash={txDatum.txHash || ''}
+                  asset={txDatum.asset}
+                  isDone={txDatum.isDone}
+                />
+              );
+            }
+            if (
+              isFuelChain(txDatum.fromNetwork) &&
+              isEthChain(txDatum.toNetwork)
+            ) {
+              return (
+                <TxListItemFuelToEth
+                  key={`${index}-${txDatum.txHash}`}
+                  txHash={txDatum.txHash || ''}
+                  asset={txDatum.asset}
+                  isDone={txDatum.isDone}
+                  date={txDatum.date}
+                />
+              );
+            }
 
-              return null;
-            })}
-          </CardList>
-        )}
-      </Card.Body>
-    </Card>
+            return null;
+          })}
+        </CardList>
+      )}
+    </>
   );
 };
 
-const styles = cssObj({
-  header: cssObj({
-    alignItems: 'center',
-    px: '$0',
-    mx: '$4',
-    borderBottom: '1px solid $border',
-  }),
-  body: cssObj({
-    minHeight: '$56',
+const styles = {
+  cardList: cssObj({
+    width: '328px',
+
+    '@md': {
+      width: '95%',
+    },
```

Comment:
this width controls are weird we should not need to use them. it's breaking consistency of sizes between bridge page and transactions page